output-failures-and-bottle-result/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Output failures summary and brew bottle result'
+description: 'Output the logs of failures summary and brew bottle result'
+
+runs:
+  using: "composite"
```

Comment:
Are quotes needed here? And if they are, can you pick one style for them? You now have `'` for the name, `"` here and nothing for the shell.

Reply:
Ah, I see so you mean about the consistency. Thank you

Reply:
Yes, it's worth checking what style the other actions use. We avoid using explicit quotes for strings unless it would introduce ambiguity. Here's an example: https://github.com/Homebrew/actions/blob/master/dismiss-approvals/action.yml

Reply:
Thanks for pointing out the examples